src/EventHandler/ConfigurableEventHandlerEntityBundle.php

```diff
@@ -51,7 +105,8 @@ public function validate() {
    * {@inheritdoc}
    */
   public function getEventNameSuffix() {
-    // Nothing to do by default.
+    $parts = explode(':', $this->pluginId);
```

Comment:
Would be good to comment the non-obvious, That's the bundle. Why not just return $this->configuration['bundle']; instead?